packages/utils/resolve-packages.js

```diff
@@ -3,7 +3,7 @@
 const path = require("path");
 const chalk = require("chalk");
 
-const modifyConfigHelper = require("./modify-config-helper");
+const modifyConfig = require("./modify-config-helper");
```

Comment:
don't see a reason to change this

Reply:
For consistency.
We're using `modifyConfig` everywhere across at codebase other than this place.

Please check this: https://github.com/webpack/webpack-cli/search?q=return+modifyHelper&unscoped_q=return+modifyHelper

Reply:
Use the previous convention 👍

Reply:
Updated all occurrences to use the previous convention.